chain/scripts/add_os_ds.sh

```diff
@@ -0,0 +1,188 @@
+bandd add-data-source \
```

Comment:
I think an absolute path didn't make sense when need to compile on other device.

DIR=`dirname "$0"`

and use path as. $DIR/../datasources/ instead same as oracle script